app/assets/js/rtMedia.backbone.js

```diff
@@ -683,6 +687,29 @@ jQuery( function( $ ) {
 					uploaderObj.uploadFiles();
 				}
 
+
+				/**
+				 *
+				 * Uploader improper enter behavior issue(124) fixed
+				 *
+				 */
+				var terms_conditions = $('#rtmedia_upload_terms_conditions');
+				if(terms_conditions.attr('id') === 'rtmedia_upload_terms_conditions'){
```

Comment:
whats use of this condition?

Reply:
Its same as `.length === 0`, for determining whether the element found or not.
Do you want me to change this to `.length === 0` condition?

Reply:
yes, please